Shared/Tests.Sync.Shared/PermissionsTests.cs

```diff
@@ -29,10 +29,10 @@
 
 using ExplicitAttribute = NUnit.Framework.ExplicitAttribute;
 
-namespace Tests.Sync.Shared
+namespace Tests.Sync
 {
     [TestFixture, Preserve(AllMembers = true)]
-    public class PermissionsTests
+    public class PermissionsTests : SyncTestBase
```

Comment:
I prefer utilizing static `Helper.Methods()` over test case inheritance.  I think it's because I like tests to be as explicit as possible, so anything that happens magically in another file is something that might cause a test to work in a way that I misunderstand. I even dislike plain `SetUp` and `TearDown` methods for this reason, but at least they are right there in the same file..
I know that we've done this before so I am not vetoing, just adding my 2c.